backend/restapi-macroservice/src/webapi/controllers/chat.controller.ts

```
@@ -55,6 +55,18 @@ export class ChatController {
+      chats: chatsResult.map((chat) => ({
+        giftCount: chat.listings.length,
+        chatId: chat.chatId,
+        chatName: chat.chatName,
+        createdAt: chat.createdAt,
+        isInterviewCompleted:
+          chat.isInterviewCompleted ||
+          (Boolean(chat.listings) && chat.listings.length > 0),
+        reasoningSummary: chat.reasoningSummary,
+      })),
+    };
     return { chats: chatsResult.map((chat) => this.mapChatToDto(chat)) };
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)